chopsticks/config.yaml

```diff
@@ -0,0 +1,17 @@
+endpoint:
+    # We fetch relevant storage elements from Spiritnet even tho we are running Peregrine (with sudo).
+    - wss://spiritnet.kilt.io
+db: db/peregrine.sqlite
+runtime-log-level: 5
+port: 50002
+wasm-override: /home/antonio/Developer/kilt-node/target/debug/wbuild/peregrine-runtime/peregrine_runtime.wasm
```

Comment:
Can we maybe use the relative path?

Reply:
Or is this file only for testing, and we delete it after?

Reply:
Yes, I will delete these once you are able to verify the correctness of the solution.